grails-plugin-controllers/src/main/groovy/grails/artefact/controller/support/RequestForwarder.groovy

```diff
@@ -108,8 +108,8 @@ trait RequestForwarder implements WebAttributes {
         webRequest.removeAttribute(UrlMappingsHandlerMapping.MATCHED_REQUEST, requestScope)
         webRequest.removeAttribute(WebUtils.ERROR_STATUS_CODE_ATTRIBUTE, requestScope)
         webRequest.removeAttribute("grailsWebRequestFilter" + OncePerRequestFilter.ALREADY_FILTERED_SUFFIX, requestScope)
-        dispatcher.forward(request, response);
         request.setAttribute(GrailsApplicationAttributes.FORWARD_ISSUED, true)
+        dispatcher.forward(request, response);
```

Comment:
So with the change to use `isForward` is there is any reason to keep this change to reverse the order? This may break existing behaviour / checks.